root/components/home/Intro.js

```diff
@@ -0,0 +1,265 @@
+/*
+ * @flow strict
+ * Copyright (C) 2022 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+import {Modal} from 'react-bootstrap';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import {faSearch, faScanner} from '@fortawesome/free-solid-svg-icons';
+
+import {reduceArtistCredit}
+  from '../../static/scripts/common/immutable-entities';
+import {ArtworkImage} from '../Artwork';
+import entityHref
+  from '../../static/scripts/common/utility/entityHref';
+
+import Blog from './Blog';
+
+const chipData = [
+  {key: 0, label: 'Artist'},
+  {key: 1, label: 'Release'},
+  {key: 2, label: 'Recording'},
+  {key: 3, label: 'Label'},
+  {key: 4, label: 'Work'},
+  {key: 5, label: 'Release Group'},
+  {key: 6, label: 'Area'},
+  {key: 7, label: 'Place'},
+  {key: 8, label: 'Annotation'},
+  {key: 9, label: 'CD Stud'},
+  {key: 10, label: 'Editor'},
+  {key: 11, label: 'Tag'},
+  {key: 12, label: 'Instrument'},
+  {key: 13, label: 'Series'},
+  {key: 14, label: 'Event'},
+  {key: 15, label: 'Documentation'},
+];
```

Comment:
Showing all of the search options by default is a bit intimidating. A bunch of these (like CD stubs, tags, annotations) are not even entities, even though the search field says "search X entities" :) I'd suggest displaying the few most relevant options by default (Artist, Label, Release maybe?) with a button to show all options.

Reply:
I am fine with a dropdown to showcase the other entities as you suggest but yeah showing them all nicely can be a good flex if we wanna, haha! :)